carmajava/launch/carma_src.launch

```diff
@@ -31,15 +31,15 @@
 -->
 <launch>
   <!-- Directory of vehicle config parameters -->
-  <arg name="vehicle_config_dir" default="$(find carma)" doc="The directory continaing vehicle config type parameters"/> <!-- TODO where to put the default -->
-  <arg name="vehicle_calibration_dir" default="$(find carma)" doc="The directory continaing vehicle calibration type parameters"/>
+  <arg name="vehicle_config_dir" default="$(find carma)../../CARMAConfig/development" doc="The directory continaing vehicle config type parameters"/>
```

Comment:
So this is assuming that `CARMAConfig` is in the same catkin workspace as the platform, correct? Just trying to make sure I understand.

Reply:
That is the default location. I tried to set all the arguments to match a development source installation. The docker_launch files change these paths for the docker install